app/classifier/feedback/feedback-summary-container.jsx

```diff
@@ -35,24 +35,30 @@ class FeedbackSummaryContainer extends React.Component {
     }, []);
     actions.feedback.setFeedback(feedbackItems);
   }
+
+  render() {
+    return (this.props.feedback.length) ? <FeedbackSummary feedback={this.props.feedback} /> : null;
+  }
 }
 
 FeedbackSummaryContainer.propTypes = {
-  actions: PropTypes.object,
+  actions: PropTypes.shape({
+    feedback: PropTypes.func
```

Comment:
`feedback` is the object containing all the actions, so `object` is the correct type expected.